README.md

```diff
@@ -8,13 +8,50 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](#Getting-Started)  
+    - [Eclipse Plugin Installation Instuctions](#Eclipse-Plugin-Installation-Instuctions)
+- [Building](#Building)
+    - [Prerequisites](#Prerequisites)
+    - [Setup Instructions](#Setup-Instructions)
+    - [Common Errors](#Common-Errors)
+- [Contributing](#Contributing)
+- [Feedback](#Feedback)
+---
 ## Getting Started
 
+### Eclipse Plugin Installation Instuctions 
+
+1. Nagivate to the [releases page](https://github.com/eclipse/lsp4jakarta/releases) and download the `*.jar` file for the Eclipse plugin. 
+
+2. Move the `*.jar` file to the **dropins** folder and start **Eclipse** in `clean` mode. 
+    - **For Mac:** 
+        - In **Finder**, navigate to the **Eclipse** application.
+        - Right-click on the application and click "Show Package Contents".
+        - Navigate through the folders as follows: **Contents > Eclipse**
+        - In this directory, update the **eclipse.ini** file by adding `-clean` to the top of this file, save, and exit the file.
+        - Next, in the same directory as the  **eclipse.ini** file, click on **dropins** folder.
+        - Move the `*.jar` inside the **dropins** folder. 
+    - **For Windows:**
+        - Locate the directory with the **Eclipse** executable. 
+        - In this directory, update the **eclipse.ini** file by adding `-clean` to the top of this file, save, and exit the file.
+        - In the same directory, click on **dropins** folder.
+        - Move the `*.jar` inside the **dropins** folder. 
+    - **For Linux:**
+        - In the terminal, use the following command to locate the directory with the **eclipse.ini** file: `ls -l /usr/bin | grep 'eclipse'`
+        - In this directory, update the **eclipse.ini** file by adding `-clean` to the top of this file, save, and exit the file.
+        - In the same directory, click on **dropins** folder.
+        - Move the `*.jar` inside the **dropins** folder. 
+
+3. Restart the **Eclipse** application.
+
+## Building
```

Comment:
I was originally thinking that the building section could be in a separate document altogether. Perhaps under `docs/Building.md`. What do you think?